test/support/models/article.ex

```diff
@@ -3,6 +3,8 @@ defmodule ExMachina.Article do
 
   schema "articles" do
     field :title, :string
+
     belongs_to :author, ExMachina.User
+    belongs_to :editor, ExMachina.User
```

Comment:
I think a comment explaining why this is here might be good. Usually I don't like comments, but without that I think it is likely someone will delete this and wonder why all the tests still pass. What do you think?